EIPS/eip-2029.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2029
+title: State Rent A - State counters contract
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: https://ethereum-magicians.org/t/eip-2029-state-counters-contract-change-a-from-state-rent-v3-proposal/3279
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-15
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A smart contract is deployed on all Ethereum networks (including testnets), with the code that simply reads the slot in its storage specified by the
+only parameter. Later, this contract becomes "special" in that Ethereum start writing state counters (number of total transactions, number of accounts,
```

Comment:
I like how [EIP 210](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-210.md) describes how to store these things. I think it is more pure to use the proposed system account here and do a transaction to said contract. Clients themselves can optimize this, but on the definition level this looks more pure to me.